circle.yml

```diff
@@ -2,7 +2,7 @@ machine:
   ruby:
     version: 2.2.3
```

Comment:
The apiaryio gem depends on it, we wouldn't be able to publish the API docs to Apiary without it.